lingua_franca/__init__.py

```diff
@@ -1,6 +1,15 @@
-from .internal import get_default_lang, set_default_lang, get_default_loc, \
-    get_active_langs, _set_active_langs, get_primary_lang_code, \
-    get_full_lang_code, resolve_resource_file, load_language, \
-    load_languages, unload_language, unload_languages, get_supported_langs
+### DO NOT CHANGE THIS IMPORT ORDER ###
```

Comment:
That's exactly what it is, that's for the duration of the feature branch until I figure out exactly where the line is.

It may be possible to reduce the scope of `internal`'s `config` imports and avoid the circle.